packages/core/src/Settings.ts

```diff
@@ -292,3 +292,4 @@ function deepSet(context, key, value) {
  * Initial Settings for the repository
  */
 Settings.getDefaultSettings().set('useCursors', true);
+Settings.getDefaultSettings().set('debug', false);
```

Comment:
what is this

Reply:
This should probably be removed, I was turning this on when debugging touch events.